File: Assignment1/submit/fourargs.py
# Copyright 2017 Pei Jia leojia@bu.edu

import sys

length = len(sys.argv)
hitPoint = 0
for i in range(1, length):
	if hitPoint < 4:
		print(sys.argv[i])
		hitPoint += 1
	else:
		print(sys.argv[i], file=sys.stderr)

File: Assignment1/fourargs.cpp
// Copyright 2017 Pei Jia leojia@bu.edu

#include <iostream>

using namespace std;

int main(int argc, char** argv)
{
	int hitPoint = 0;
	for(int i=1; i<argc; i++){
		if(hitPoint<4){
			cout << argv[i] << "\n";
			hitPoint++;
		}else{
			cerr << argv[i] << "\n";
		}
	}
	return 0;
}
